docs/mkdocs.yml

```diff
@@ -579,6 +579,7 @@ nav:
         - web remove: 'cmd/spo/web/web-remove.md'
         - web roleinheritance reset: 'cmd/spo/web/web-roleinheritance-reset.md'
         - web set: 'cmd/spo/web/web-set.md'
+        - web roleinheritance break: 'cmd/spo/web/web-roleinheritance-break.md'
```

Comment:
You are referencing an empty documentation page here. Maybe you have forgotten to fill this file with content. Could you add the documentation to the file please? An example can be found here: https://github.com/pnp/cli-microsoft365/blob/main/docs/docs/cmd/spo/list/list-roleinheritance-break.md